### ISSUE TITLE
Simplify the installation

### ISSUE THREAD

**Issue by AssabTigleG** [flutter/uxr#135]
- Make a one step process, in which Flutter automates its installation
- or
- Make it a little friendly

**Comment by AliYar-Khan** [comment]
agree. Like a power shell script for windows, bash script for linux etc. This will help alot.
